src/builders/skopeo_builder.rs

```diff
@@ -460,6 +464,15 @@ impl SkopeoSyslinuxBuilder {
             .context("Failed to install rootfs from built container")
     }
 
+    /// Create `/workspace` directory in the VM, which will be used as output mountpoint.
+    fn create_workspace() -> Result<()> {
+        let ws_path = env::temp_dir().join("mnt").join("workspace");
```

Comment:
Is it intentional that the comment has "Create `/workspace`" but in reality the code creates `/tmp/mnt/workspace` ?

Reply:
I think the comment says "Create /workspace in the VM" and `env::temp_dir().join("mnt")` is just a very bad name for the VM image root directory

Reply:
Exactly. We create `/tmp/mnt/workspace` and since `/tmp/mnt` is a mount point for VM root fs, it means we create "`/workspace` inside VM".